src/clib/lib/enkf/enkf_obs.cpp

```diff
@@ -460,6 +460,10 @@ static void handle_history_observation(enkf_obs_type *enkf_obs,
                         "** Could not load historical data for observation:%s "
                         "- ignored\n",
                         obs_key);
+
```

Comment:
Cant comment above this, but should be possible to add:     if (num_hist_obs > 0 && enkf_obs->refcase == NULL) 
```cpp
{
        stringlist_free(hist_obs_keys);
        throw exc::invalid_argument("REFCASE required for SUMMARY_OBSERVATION");
    }
```
at line 421, and catch it in `EnKFMain` and convert it to `ConfigValidationError` to make it shown up in the Suggester

Reply:
Idealy we should probably initalize the observations outside `EnKFMain`, but that is outside the scope here.

Reply:
I've updated the code and raised a `ConfigValidationError` from python side.